README.md

```diff
@@ -19,39 +19,13 @@
 
 ## Quick Start
 
-1. Copy docker-compose.yml.example to docker-compose.yml and update as needed. See example below:
-[Docker-compose](https://docs.docker.com/compose/install/) example:
+Using [Docker-compose](https://docs.docker.com/compose/install/):
 
-```yaml
-version: "3"
-
-# More info at https://github.com/pi-hole/docker-pi-hole/ and https://docs.pi-hole.net/
-services:
-  pihole:
-    container_name: pihole
-    image: pihole/pihole:latest
-    # For DHCP it is recommended to remove these ports and instead add: network_mode: "host"
-    ports:
-      - "53:53/tcp"
-      - "53:53/udp"
-      - "67:67/udp" # Only required if you are using Pi-hole as your DHCP server
-      - "80:80/tcp"
-    environment:
-      TZ: 'America/Chicago'
-      # WEBPASSWORD: 'set a secure password here or it will be random'
-    # Volumes store your data between container upgrades
-    volumes:
-      - './etc-pihole:/etc/pihole'
-     # - './etc-dnsmasq.d:/etc/dnsmasq.d' # Only needed if you have some custom configs for dnsmasq
-    # https://github.com/pi-hole/docker-pi-hole#note-on-capabilities
-    cap_add:
-      - NET_ADMIN # Required if you are using Pi-hole as your DHCP server, else not needed
-    restart: unless-stopped
-```
+1. Copy [the example docker compose file](/examples/docker-compose-example.yml) and update as needed
```

Comment:
I typically find it nicer if READMEs have the `docker compose` example right there because you can find it more easily when skimming over it.

Reply:
Restored

Reply:
👍